SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/UserIdEventArgs.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Microsoft.AppCenter.Windows.Shared.Utils
+{
+    /// <summary>
+    /// Event args for event that occurs when an user change is received.
+    /// </summary>
+    public class UserIdUpdatedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// User Id.
+        /// </summary>
+        public string UserId
```

Comment:
We usually inline that when there is no code for getter/setter

`public string UserId { get; internal set; }`